service/controller/resource/heartbeat/resource.go

```diff
@@ -100,10 +102,13 @@ func toHeartbeat(v interface{}, installation string, pipeline string) (*heartbea
 
 func hasChanged(current, desired heartbeat.Heartbeat) bool {
 	// Ignore those fields for comparison by setting them to the same value.
-	current.Enabled = true
-	desired.Enabled = true
 	current.Expired = true
 	desired.Expired = true
 
+	// We get the ID back from opsgenie
```

Comment:
Is this to effectively ignore the `id` in comparison?

Reply:
Yes, I am trying another approach though